orderbook/openapi.yml

```diff
@@ -204,6 +204,42 @@ paths:
                 type: array
                 items:
                   $ref: "#/components/schemas/Order"
+  /api/v1/fee:
+    get:
+      description: |
+        The fee that is charged for placing an order.
+        The fee is described by a minimum fee - in order to cover the gas costs for onchain settling - and
+        a feeRatio charged to the users for using the service.
+      parameters:
+        - name: sellToken
+          in: query
+          schema:
+            $ref: "#/components/schemas/Address"
+          required: true
+        - name: buyToken
+          in: query
+          schema:
+            $ref: "#/components/schemas/Address"
+          required: true
+        - name: amount
+          in: query
+          schema:
+            $ref: "#/components/schemas/TokenAmount"
+          required: true
```

Comment:
1. What is this amount's unit? Integer atoms or decimal units
2. Is it the sell amount or buy amount? I assume it matches the kind (sell amount for sell order, etc)

Reply:
Yes, it would be the same as in the order endpoint for now. For convenience it would probably be nice to let people flip-flop between base units and atoms but for now we don't have token info in the backend.

Reply:
seconding allowing flip flopping, can obv be later no issue there